Source/WebCore/rendering/style/BasicShapes.h

```diff
@@ -203,9 +214,10 @@ class BasicShapeCircle final : public BasicShape {
     BasicShapeCenterCoordinate m_centerX;
     BasicShapeCenterCoordinate m_centerY;
     BasicShapeRadius m_radius;
+    bool centerWasOmitted = false;
```

Comment:
Removed this. I don't think there's a good way to use std::optional here since the code does the conversion from nullptr to the default values for the center of the circle and ellipse shape. This is a special case for motion path specifically so I think it's simpler to use a bool here.